src/main/kotlin/no/nav/syfo/cronjob/VerifyBehandlereForKontorCronjob.kt

```diff
@@ -29,8 +29,9 @@ class VerifyBehandlereForKontorCronjob(
 ) : DialogmeldingCronjob {
     private val runAtHour = 6
 
-    override val initialDelayMinutes: Long =
-        calculateInitialDelay("VerifyBehandlereForKontorCronjob", runAtHour)
+    override val initialDelayMinutes: Long = 2
```

Comment:
Disse endringene skal rulles tilbake etterpå - de her er bare for å trigge oppdatering av kontoret vi har problemer med så vi kan få den saken ut av verden uten å vente to dager til.